content/v2.0/organizations/members/add-member.md

```diff
@@ -7,25 +7,33 @@ menu:
     name: Add a member
     parent: Manage members
 weight: 201
-draft: true
 ---
 
-Use the InfluxDB user interface (UI) or the `influx` command line interface (CLI)
-to add a member to an organization.
+Use the `influx` command line interface (CLI) to add a member to an organization.
 
-## Add a member to an organization in the InfluxDB UI
+{{% cloud-msg %}}
+Adding members is currently unavailable in InfluxDB Cloud 2.
```

Comment:
InfluxDB Cloud 2.0 (to be consistent w other docs)

Reply:
Or just the `{{< cloud-name >}}` shortcode.